### PR TITLE
backport: trivial backports 2023 04 17

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
backport via script on April 17th

## What was done?
Trivial backports via script

## How Has This Been Tested?
make check and functional tests

## Breaking Changes
Should be none

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: doc/fuzzing.md [hunk 1]
```diff
@@ -83,6 +83,10 @@ INFO: seed corpus: files: 991 min: 1b max: 1858b total: 288291b rss: 150Mb
 …
 ```
 
+## Run without sanitizers for increased throughput
+
+Fuzzing on a harness compiled with `--with-sanitizers=address,fuzzer,undefined` is good for finding bugs. However, the very slow execution even under libFuzzer will limit the ability to find new coverage. A good approach is to perform occasional long runs without the additional bug-detectors (configure `--with-sanitizers=fuzzer`) and then merge new inputs into a corpus as described in the qa-assets repo (https://github.com/bitcoin-core/qa-assets/blob/main/.github/PULL_REQUEST_TEMPLATE.md).  Patience is useful; even with improved throughput, libFuzzer may need days and 10s of millions of executions to reach deep/hard targets.
```
Comment (knst) [inline doc/fuzzing.md:88]: https://github.com/bitcoin-core/qa-assets/blob/main/.github/PULL_REQUEST_TEMPLATE.md
Should we dashificate this url to
https://github.com/dashpay/dash/blob/master/.github/PULL_REQUEST_TEMPLATE.md
(not exactly same, but at least it is not bitcoin-core)

**Review by knst — CHANGES_REQUESTED** [review]
LGTM except a dashification for doc/fuzzing.md

File: doc/fuzzing.md (hunk 1 above)
Comment (PastaPastaPasta) [reply doc/fuzzing.md:88]: Meh, maybe let's not for now

Comment (PastaPastaPasta) [reply doc/fuzzing.md:88]: let's dashify once we maybe actually use fuzzing

**Review by knst — DISMISSED** [review]
> let's dashify once we maybe actually use fuzzing

utACK

**Review by UdjinM6 — APPROVED** [review]
utACK

**Comment by UdjinM6** [comment]
ci failed to start for some reason but tests pass locally, merging

**Pull request merged**